components/controls/ControlsCenter.tsx

```diff
@@ -5,22 +5,23 @@ import MicrophoneButton from "./buttons/MicrophoneButton";
 import CameraButton from "./buttons/CameraButton";
 import ScreenShareButton from "./buttons/ScreenShareButton";
 import SettingsButton from "./buttons/SettingsButton";
+import ChatButton from "./buttons/ChatButton";
+import useWindowDimensions from "hooks/useWindowDimension";
 
 interface Props {
   onLeave: () => void;
-  onRename: (newName: string) => void;
 }
 
-export default function ControlsCenter({
-  onLeave,
-  onRename,
-}: Props): JSX.Element {
+export default function ControlsCenter({ onLeave }: Props): JSX.Element {
+  const { width = 0, height = 0 } = useWindowDimensions();
+
   return (
-    <HStack>
+    <HStack spacing="24px">
       <MicrophoneButton />
       <CameraButton />
-      <ScreenShareButton />
-      <SettingsButton onLeave={onLeave} onRename={onRename} />
+      {!!navigator.mediaDevices.getDisplayMedia && <ScreenShareButton />}
```

Comment:
This is interesting to check for `getDisplayMedia`. I generally prefer to keep any in calls to media methods like this in our hooks. My reasoning is that if we want to open source those hooks eventually it should be possible to use the hooks without also learning the specifics of the media APIs. Not a big deal though.

Reply:
Yeah, we could have a `screenShareSupported` from the `useSpace` hook, and restrict usage of the screen sharing behind that as well.

Reply:
I added a `isLocalScreenShareSupported` as a part of the `useSpace` hook and I'm using it here.

Reply:
Cool, thanks!